packages/ember-glimmer/tests/unit/template-factory-test.js

```diff
@@ -55,7 +55,8 @@ moduleFor('Template factory test', class extends RenderingTest {
 
     this.render('{{x-precompiled name="precompiled"}} {{x-compiled name="compiled"}}');
 
-    assert.equal(env._templateCache.misses, 2, 'misses 2');
+    // This extra miss is from rendering the root layout in the test harness component
+    assert.equal(env._templateCache.misses, 3, 'misses 3');
```

Comment:
this is annoying, the test harness could use component.layoutName instead and register the template